db/migrate/20210119162528_delete_puppet_and_ostree_repos.rb

```diff
@@ -34,7 +41,7 @@ def up
         FakeContentViewPuppetEnvironment.delete_all
         FakePuppetModule.delete_all
 
-        Katello::Repository.puppet_type.delete_all
+        ::Katello::Repository.delete(puppet_repositories)
```

Comment:
shouldn't this be puppet_repositories.delete_all?

Reply:
or does find_by_sql return an array?   I'm not actual familiar with delete() and passing in objects

Reply:
Yeah I had an issue with that. `find_by_sql` was returning an array.